src/ResourceManager/Resources/Commands.Resources/help/New-AzureRmResourceLock.md

```diff
@@ -185,7 +185,7 @@ Accept wildcard characters: False
 
 ### -LockLevel
 Specifies the level for the lock.
-Currently, the only valid value is CanNotDelete.
+Currently, valid values are CanNotDelete, ReadOnlyLock.
```

Comment:
@simon-michaud using the latest version of `AzureRM.Resources`, it looks like `ReadOnly` is a valid value, not `ReadOnlyLock`. Is there somewhere else that you're seeing `ReadOnlyLock` as a valid value?

Reply:
Right, made a typo, fixed it